helix-term/src/ui/mod.rs

```diff
@@ -382,6 +382,13 @@ pub mod completers {
         })
     }
 
+    pub fn variables(_: &Editor, input: &str) -> Vec<Completion> {
+        fuzzy_match(input, helix_view::editor::VARIABLES, false)
+            .into_iter()
```

Comment:
Would the available variables ever change depending on the state of Editor? I think this can drop the `&Editor` parameter

Reply:
It looks like this hasn't been resolved yet

Reply:
You mean remove the `&Editor` parameter from the function? I can't do it without breaking the `pub type Completer = fn(&Editor, &str) -> Vec<Completion>` requirement

Reply:
Ah right good catch. Let's leave it as-is :+1: